src/test/java/com/jcabi/github/mock/MkCommentTest.java

```diff
@@ -90,14 +91,76 @@ public void canCompareInstances() throws Exception {
     }
 
     /**
+     * MkComment should store all its data properly.
+     * We should get the proper data back when accessing its properties.
+     * @throws Exception when a problem occurs.
+     */
+    @Test
+    public void dataStoredProperly() throws Exception {
+        final String cmt = "what's up?";
+        final long before = this.now();
+        final Comment comment = this.comment(cmt);
+        final long after = this.now();
+        MatcherAssert.assertThat(
+            comment.number(),
+            Matchers.greaterThan(0)
+        );
+        final Comment.Smart smart = new Comment.Smart(comment);
+        MatcherAssert.assertThat(
+            smart.issue().number(),
+            Matchers.greaterThan(0)
+        );
+        MatcherAssert.assertThat(
+            smart.author().login(),
+            Matchers.equalTo("jeff")
+        );
+        MatcherAssert.assertThat(
+            smart.body(),
+            Matchers.equalTo(cmt)
+        );
+        MatcherAssert.assertThat(
+            smart.url(),
+            // @checkstyle LineLength (1 line)
+            Matchers.equalTo(new URL("https://api.jcabi-github.invalid/repos/jeff/test/issues/comments/1"))
+        );
+        MatcherAssert.assertThat(
+            smart.createdAt().getTime(),
+            Matchers.greaterThanOrEqualTo(before)
+        );
+        MatcherAssert.assertThat(
+            smart.createdAt().getTime(),
+            Matchers.lessThanOrEqualTo(after)
+        );
+        MatcherAssert.assertThat(
+            smart.updatedAt().getTime(),
+            Matchers.greaterThanOrEqualTo(before)
+        );
+        MatcherAssert.assertThat(
+            smart.updatedAt().getTime(),
+            Matchers.lessThanOrEqualTo(after)
+        );
+    }
+
+    /**
      * Create a comment to work with.
+     * @param text Text of comment
      * @return Comment just created
      * @throws Exception If some problem inside
      */
-    private Comment comment() throws Exception {
+    private Comment comment(final String text) throws Exception {
         return new MkGithub().repos().create(
             Json.createObjectBuilder().add("name", "test").build()
-        ).issues().create("hey", "how are you?").comments().post("what's up?");
+        ).issues().create("hey", "how are you?").comments().post(text);
     }
 
+    /**
+     * Obtains the current time.
+     * @return Current time (in milliseconds since epoch) truncated to the nearest second
+     */
+    private long now() {
+        final int mspersec = 1000;
+        final long sinceepoch = new Date().getTime();
+        final long milliseconds = sinceepoch % mspersec;
+        return sinceepoch - milliseconds;
```

Comment:
@cvrebert all these things can be done in one line
```java
return new Date().getTime() - sinceepoch % 1000;
```
without any one-time-variable.
The code is still understandable. Javadoc explains what is going on.

By the way, why do we need this truncation?